packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.css

```diff
@@ -2,6 +2,9 @@
     display: grid;
     grid-template-columns: 1fr;
     gap: 1rem;
+
+    position: sticky;
```

Comment:
I somewhat expected this to behave oddly on mobile, but it appears the stickiness just never kicks in. I did observe some odd behavior when toggling `position: sticky` on and off; consider explicitly setting back to `position: relative` and `top: auto` in mobile rule.

![](https://gyazo.com/ccca1be7558d936be288e843458efe1b.gif)

Reply:
I wasn't seeing anything happen when I toggled the sticky position but I think it's fine to reset to default on mobile where the styles no longer apply.